tests/foreman/ui/test_lifecycleenvironment.py

```diff
@@ -154,3 +163,86 @@ def test_positive_create_environment_after_host_register(self):
 
         @caseautomation: notautomated
         """
+
+    @skip_if_bug_open('bugzilla', 1420511)
+    @run_only_on('sat')
+    @tier2
+    def test_positive_lvce_user_access(self):
+        """As a user attempt to view a lifecycle environment created
+        by admin user
+
+        @id: 768b647b-c530-4eca-9caa-38cf8622f36d
+
+        @BZ: 1420511
+
+        @steps:
+        with super admin user
+
+        1. Create an additional lifecycle environments other than Library
+        2. create a user
+        3. Create a role with the the following permissions:
+
+        * (Miscellaneous): access_dashboard
+        * Lifecycle Environment:
+
+          1. edit_lifecycle_environments
+          2. promote_or_remove_content_views_to_environment
+          3. view_lifecycle_environments
+
+        * Location: view_locations
+        * Organization: view_organizations
+
+        4. Assign the created role to the created user
+        5. Login as the created user
+        6.  Content -> Lifecycle Environments
+
+        @Assert: The additional lifecycle environment is view-able
+        and accessible by the created user.
+
+        @CaseLevel: Integration
+        """
+        role_name = gen_string('alpha')
+        env_name = gen_string('alpha')
+        user_login = gen_string('alpha')
+        user_password = gen_string('alpha')
+        org = entities.Organization().create()
+        role = entities.Role(name=role_name).create()
+        permissions_types_names = {
+            None: ['access_dashboard'],
+            'Organization': ['view_organizations'],
+            'Location': ['view_locations'],
+            'Katello::KTEnvironment': [
+                'view_lifecycle_environments',
+                'edit_lifecycle_environments',
+                'promote_or_remove_content_views_to_environments'
+            ]
+        }
+        create_role_permissions(role, permissions_types_names)
+        entities.User(
+            default_organization=org,
+            organization=[org],
+            role=[role],
+            login=user_login,
+            password=user_password
+        ).create()
+        # create a life cycle environment as admin user and ensure it's visible
+        with Session(self.browser) as session:
+            make_lifecycle_environment(
+                session, org=org.name, name=env_name)
+            self.assertIsNotNone(self.lifecycleenvironment.search(env_name))
+
+        # ensure the created user also can find the created life cycle
+        # environment link
+        with Session(self.browser, user=user_login, password=user_password):
+            # to ensure that the created user has only the assigned
+            # permissions, check that hosts menu tab does not exist
+            self.assertIsNone(
+                self.content_views.wait_until_element(
+                    menu_locators['menu.hosts'], timeout=5)
```

Comment:
5 secs is too much for element which should not appear. Please set 1 sec instead.